mobile/lib/main.dart

```diff
@@ -1,7 +1,27 @@
+import 'package:polydodo/protos/helloworld.pbgrpc.dart';
+import 'package:grpc/grpc.dart';
 import 'package:flutter/material.dart';
 
+import 'protos/helloworld.pb.dart';
 import 'src/app.dart';
 
-void main() {
+void main() async {
+  // GRPC TEST
+  final channel = ClientChannel(
+    '10.0.2.2', // Emulator alias to host's loopback interface (when server is running on computer)
```

Comment:
On ne doit jamais hardcode les adresses. On peut utiliser les dart defines à la place:

https://dartcode.org/docs/using-dart-define-in-flutter/